src/main/java/org/apache/datasketches/theta/Sketches.java

```diff
@@ -79,15 +79,32 @@ public static int getMaxAnotBResultBytes(final int maxNomEntries) {
   }
 
   /**
-   * Ref: {@link Sketch#getMaxCompactSketchBytes(int)}
-   * @param numberOfEntries  Ref: {@link Sketch#getMaxCompactSketchBytes(int)},
-   * {@code numberOfEntries}
-   * @return Ref: {@link Sketch#getMaxCompactSketchBytes(int)}
-   */
+   * Returns the maximum number of storage bytes required for a CompactSketch with the given
+   * number of actual entries. Note that this assumes the worse case of the sketch in
```

Comment:
worse --> worst

Reply:
Didn't even mean to pull this in. I think I inadvertently pulled form the main branch -- I thought I fetched and switches to the quantiles-filter branch! But will fix.